python/cudf_polars/tests/test_quent.py

```
@@ -0,0 +1,534 @@
+
+
+@pytest.fixture(params=["ray", "dask", "spmd"])
+def engine_with_quent_context(
+    request: pytest.FixtureRequest,
+    quent_context: QuentContext,
+    streaming_engine_factory: Callable[..., StreamingEngine],
+) -> StreamingEngine:
+    backend = request.param
+    if backend == "ray":
+        pytest.importorskip("ray")
+        import cudf_polars.experimental.rapidsmpf.frontend.ray
```

Ruff (0.15.12)

[warning] `import` should be at the top-level of a file

(PLC0415)